cmd/version.go

```diff
@@ -18,15 +18,16 @@ package main
 
 import (
 	"fmt"
+	"runtime"
 )
 
 // var needs to be used instead of const as ldflags is used to fill this
 // information in the release process
 var (
 	kubeBuilderVersion      = "unknown"
 	kubernetesVendorVersion = "unknown"
-	goos                    = "unknown"
-	goarch                  = "unknown"
+	goos                    = runtime.GOOS
+	goarch                  = runtime.GOARCH
```

Comment:
The info will be filled, see:

> $ kubebuilder version
Version: main.version{KubeBuilderVersion:"v2.0.0-1506-g7e83b313-dirty", KubernetesVendor:"unknown", GitCommit:"7e83b31354bab9d3d875f36790d0e3f33ca32c8e", BuildDate:"2022-07-04T19:01:54Z", GoOs:"darwin", GoArch:"amd64"}

So, could you please let us know more about the motivation for this one?